src/Microsoft.AspNetCore.Razor.Language/DefaultRazorTagHelperBinderPhase.cs

```diff
@@ -8,24 +8,23 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    internal class TagHelperBinderSyntaxTreePass : IRazorSyntaxTreePass
+    internal class DefaultRazorTagHelperBinderPhase : RazorEnginePhaseBase, IRazorTagHelperBinderPhase
```

Comment:
From a naming perspective I don't see the value in having `Razor` in this name. Do our other phases follow the inclusion of Razor?